spring-cloud-gcp-data-firestore/pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
+         xmlns="https://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="https://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>spring-cloud-gcp</artifactId>
+        <groupId>org.springframework.cloud</groupId>
+        <version>1.2.0.BUILD-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>spring-cloud-gcp-data-firestore</artifactId>
+    <name>Spring Cloud GCP Firestore Module</name>
+    <description>Spring Cloud GCP Firestore Module</description>
+
+    <properties>
+        <reactor.version>Californium-SR6</reactor.version>
+        <grpc.version>1.20.0</grpc.version>
```

Comment:
We shouldn't have any dependency versions in this pom. It's all managed in the parent pom.